libs/main/main-window/src/sidebar/sidebar.component.html

```diff
@@ -539,6 +539,22 @@
             </a>
             <div class="sidebar-submenu" [@slide]="menuStates['texts']">
               <ul>
+                <li>
+                  <a href="#" routerLinkActive="active" [routerLink]="'texts/select-acore-string'">
+                    {{ 'SIDEBAR.TEXTS.SELECT_ACORE_STRING_TEXT' | translate }}
+                  </a>
+                </li>
+                @if (broadcastTextHandlerService.selected) {
+                  <li>
+                    <a href="#" routerLinkActive="active" [routerLink]="'texts/acore-string'">
+                      {{ 'SIDEBAR.TEXTS.ACORE_STRING_TEXT' | translate }}
+                      @if (broadcastTextHandlerService.isUnsaved) {
```

Comment:
@Exitare  I've found a small bug while testing: this should be its own handler service and not the one from `broadcastTextHandlerService`